scripts/blocks-utils.js

```diff
@@ -445,6 +445,25 @@ function getCurrentHost() {
   return hostUrl;
 }
 
+/**
+ * Util function to append no results message in the block with no data to display
+ * @param {*} block which has no data to display
+ * @param {*} containerSelector selector for the container to append no results message
+ */
+const handleNoResults = (block, containerSelector) => {
```

Comment:
@vivesing  can this method directly take the element where we need to show 'no records found'. IMO we can remove the block div requirement here and just take directly the container where we need to update the content ?

Reply:
done, talking element now

Reply:
LGTM